drivers/include/at86rf2xx.h

```diff
@@ -167,6 +167,11 @@ typedef struct {
     uint8_t pending_tx;                 /**< keep track of pending TX calls
                                              this is required to know when to
                                              return to @ref at86rf2xx_t::idle_state */
+/* Only radios with the XAH_CTRL_2 register support frame retry reporting */
+#ifdef AT86RF2XX_XAH_CTRL_2__ARET_FRAME_RETRIES_OFFSET
```

Comment:
Murdock fails, I guess this files missed `#include "at86rf2xx_registers.h"`

Reply:
Oh, of course :(

Reply:
I think it would be better to leave this at if MODULE_AT86RF233 etc, instead of including the internal header in the external API declarations.